### ISSUE TITLE
Enter new Zettel name only once with org-roam-insert

### ISSUE THREAD

**Issue by chrispoole643** [jethrokuan/org-roam#53]
A feature request really for anyone who can more quickly think how to do this...

When I invoke `org-roam-insert` and type a new name for a Zettel that doesn't yet exist, it drops me to a prompt to enter the File ID (`org-roam--insert-title`). Feature request for it to take whatever name I've just entered, a file that doesn't yet exist, and create that file with that name.

(Similar to how Deft functions with `deft-new-file`.)

**Comment by jethrokuan** [comment]
Done!

**Comment by chrispoole643** [comment]
Wow: quick! Don't think I can reopen it but tried it out and don't think it's doing quite what I intended.

If I run `org-roam-insert`, and enter a file title that isn't in the list (I'm using ivy here, for completion support...), the `[[title]]` in my Org file will be populated, but it still prompts for `Enter new file id: `. I want the file ID to be auto-populated too, as something like `{{title without special chars to work on all filesystems.org}}`. Preserve case, spaces, and perhaps remove the rest?

(I tried looking for a builtin elisp function to do this in a stadard Emacs way but can't find much...)
